src/main/java/org/gridsuite/securityanalysis/server/SecurityAnalysisController.java

```diff
@@ -151,4 +152,19 @@ public ResponseEntity<Mono<Void>> stop(@Parameter(description = "Result UUID") @
         Mono<Void> result = service.stop(resultUuid, receiver);
         return ResponseEntity.ok().body(result);
     }
+
+    @GetMapping(value = "/providers", produces = APPLICATION_JSON_VALUE)
+    @Operation(summary = "Get all security analysis providers")
+    @ApiResponses(value = {@ApiResponse(responseCode = "200", description = "Security analysis providers have not been found")})
+    public ResponseEntity<List<String>> getProviders() {
+        return ResponseEntity.ok().contentType(MediaType.APPLICATION_JSON)
+                .body(service.getProviders());
+    }
+
+    @GetMapping(value = "/default-provider", produces = TEXT_PLAIN_VALUE)
+    @Operation(summary = "Get security analysis default provider")
+    @ApiResponses(@ApiResponse(responseCode = "200", description = "The security analysis default provider has not been found"))
```

Comment:
The security analysis default provider has been found

Reply:
Done